config/locales/en.yml

```diff
@@ -201,28 +201,6 @@ en:
     sign_in: List a teaching job
     sign_out: Sign out
 
-  nqt_job_alerts:
```

Comment:
I'm sure I'm missing something but where did this go?

Reply:
I don't think it's used anymore. I think initially it was used for the confirmation page after an nqt job alert was created? I checked the confirmation page for normal job alerts and nqt job alerts after deleting it and there wasn't any content missing!